File: Bubble.py
def bubble_sort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]

arr = [45, 37, 22, 92, 0, 10, 90]
bubble_sort(arr)
print("Sorted array:")
for i in range(len(arr)):
    print(arr[i], end=" ")
print()

File: Bubble.js
function bubbleSort(arr) {
    let n = arr.length;
    for (let i = 0; i < n; i++) {
        for (let j = 0; j < n - i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                // Swap arr[j] and arr[j+1]
                let temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }
}

let arr = [45, 37, 22, 92, 0, 10, 90];
bubbleSort(arr);
console.log("Sorted array:");
for (let i = 0; i < arr.length; i++) {
    console.log(arr[i] + " ");
}
console.log();
